test/BitwiseTest.php

```diff
@@ -177,4 +177,53 @@ public function testSetStrict()
         $validator = new Bitwise(0x1, Bitwise::OP_AND, true);
         $this->assertTrue($validator->getStrict());
     }
+
+    public function testConstructWithArguments()
+    {
+        $control = 0x1;
+        $operator = Bitwise::OP_AND;
+        $strict = true;
+
+        $validator = new Bitwise($control, $operator, $strict);
+
+        $this->assertEquals($control, $validator->getControl());
+        $this->assertEquals($operator, $validator->getOperator());
+        $this->assertEquals($strict, $validator->getStrict());
+    }
+
+    public function testGetControl()
+    {
+        $control = 0x1;
+        $validator = new Bitwise($control, Bitwise::OP_AND, false);
+        $this->assertEquals($control, $validator->getControl());
+    }
+
+    public function testGetOperator()
+    {
+        $operator = Bitwise::OP_AND;
+        $validator = new Bitwise(0x1, $operator, false);
+        $this->assertEquals($operator, $validator->getOperator());
+    }
+
+    public function testGetStrict()
+    {
+        $strict = true;
+        $validator = new Bitwise(0x1, Bitwise::OP_AND, $strict);
+        $this->assertEquals($strict, $validator->getStrict());
+    }
+
+    public function testIsValidWithInvalidOperator()
+    {
+        $validator = new Bitwise(0x1, 'or', false);
+        $expectedResult = false;
+        $this->assertEquals($expectedResult, $validator->isValid(0x2));
```

Comment:
I'd argue you should use either `assertSame($expectedResult, $validator->isValid(...))` or, more simply, `assertFalse($validator->isValid(...))`. I can make that change on merge, however.